### PR TITLE
multicall contract added to stylus by example

### PR DESCRIPTION
**Pull request by GreatSoshiant**

### CODE REVIEW

**Comment by Jason-W123** [comment]
Thanks! @GreatSoshiant , since this is an application example, maybe we can also add this to [example code]https://github.com/OffchainLabs/stylus-by-example/tree/master/example_code?

**Comment by GreatSoshiant** [comment]
> Thanks! @GreatSoshiant , since this is an application example, maybe we can also add this to [example code](https://github.com/OffchainLabs/stylus-by-example/tree/master/example_code)?

File: example_code/applications/multi_call/licenses/DCO.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Developer Certificate of Origin
```
Comment (TucksonDev) [inline example_code/applications/multi_call/licenses/DCO.txt:1]: I think we should remove the `licenses` folder

File: example_code/applications/multi_call/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+[package]
+name = "stylus-multi-call-contract"
+version = "0.1.5"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+keywords = ["arbitrum", "ethereum", "stylus", "alloy"]
+description = "Stylus multi call example"
+
+[dependencies]
+alloy-primitives = "0.3.1"
```
Comment (TucksonDev) [inline example_code/applications/multi_call/Cargo.toml:10]: With the latest cargo stylus, this config does not compile. But if we change allow-primites and allow-sol-types to 0.7.6 and the stylus-sdk to 0.5.2 it builds correctly.

If it's the same case for you, can you update the config here?

File: src/app/applications/multi_call/page.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  description:
+    "An example implementation of the Multi Call contract in Rust using Arbitrum Stylus.",
+};
+
+{/* Begin Content */}
+
+# Multicall
+
+An Arbitrum Stylus version implementation of [Solidity Multi Call contract](https://solidity-by-example.org/app/multi-call/) that aggregates multiple queries using a for loop and RawCall.
+
```
Comment (TucksonDev) [inline src/app/applications/multi_call/page.mdx:12]: In all other examples, we include the interface of the contract. We should include it here for consistency.

**Review by TucksonDev — DISMISSED** [review]
Looking good!
A few changes requested on the page and the config file.

**Pull request merged**